governance/working groups/working-group-charter.md

```diff
@@ -0,0 +1,38 @@
+# Working group charter template
+
+## Overview
+
+This section should summarise in one or two paragraphs the Working group proposal, for any community member to read and quickly grasp the essence of the WG.
+
+## Background
+
+This section should dive into more detail on what the WG aims to address, why it is undertaking this work, and reference any background material/pre-existing work that this proposal builds on.
+
+## Target outputs
+This section should contain the target outputs of the WG
+
+## Meeting mechanisms
+This section should include details on how, where and when the WG intends to meet
+
+## Communication mechanisms
+This section should outline how the WG intends to communicate progress with its members, the UK TRE Community more widely, and any other interested stakeholders.
+
+## Working Group roles
+This section should outline the different roles within the Working Group. Examples include:
+- **Chairs**: Leaders of the WG
+- **Contact**: Primary contact for the WG
+- **Participating members**: Members who are undertaking work within the Working Group
+
+## Getting involved
+
+This section should detail how interested parties can get involved with joining the WG.
+
+It should also detail expected weekly time commitment for defined roles above
+
+## Agreement
+*Please submit this charter with all boxes checked below*
+
+By submitting this charter, this working group agrees to:
+- [ ] Share any outputs openly under a [CC BY 4.0 licence](https://creativecommons.org/licenses/by/4.0/)
```

Comment:
Do we want to be this specific? Could alternatively say any CC license?

What about software outputs?

Reply:
Some funding agencies will state which open licenses are permissable. Can we just say "an open licence (e.g. open-source, creative commons, or similar)" for now?

Reply:
For software I think it is fairly safe to say "any OSI approved" or "any FSF approved" license.

Not sure if there is an equivalent for creative licenses.
I could see not allowing licenses which restrict who can use the outputs (_e.g._ non-commercial or ethical licenses).